source/views/streaming/radio/index.js

```diff
@@ -165,12 +171,22 @@ export default class KSTOView extends React.PureComponent<void, Props, State> {
 
             <View style={buttonStyles.space} />
 
-            <ScheduleButton
+            <SmallActionButton
+              icon="ios-call"
```

Comment:
This icon should be changed per-platform.

Reply:
Agreed.  It works in my Android emulator, but it'd be nice to be platform-specific or general.